seacrowd/sea_datasets/ac_iquad/ac_iquad.py

```diff
@@ -0,0 +1,230 @@
+            schema="source",
+            subset_id=f"{_DATASETNAME}_complex",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_complex_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_complex",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "question": datasets.Value("string"),
+                    "sparql": datasets.Value("string"),
+                    "answer": datasets.Value("string"),
+                    "context": datasets.Value("string"),
+                    "answerline": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.qa_features
+
+            if self.config.subset_id.split("_")[2] == "complex":
+                features["meta"] = {"sparql": datasets.Value("string"), "answer_meta": datasets.Value("string"), "type": datasets.Value("string")}
+
+            else:
+                features["meta"] = {"sparql": datasets.Value("string"), "answer_meta": datasets.Value("string")}
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        subset = self.config.name.split("_")[2]
+        data_dir = dl_manager.download_and_extract(_URLS[_DATASETNAME])
+
+        if subset == "simple":
+            subset = "single"
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, f"{subset}_train.json"),
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, f"{subset}_test.json"),
+                    "split": "test",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        with open(filepath, "r", encoding="utf-8") as file:
+            data_json = json.load(file)
+
+        df = pd.json_normalize(data_json)
+
+        for index, row in df.iterrows():
+
+            if self.config.schema == "source":
+                example = row.to_dict()
```

Comment:
It throws an error since the `tipe` field isn't introduced in `_info` feature list for source

Reply:
Yup, I forgot that case before; but done in https://github.com/SEACrowd/seacrowd-datahub/pull/641/commits/7e8eb1b1448fa89748414a36d38a35346705524b commit.